lib/loader.js

```diff
@@ -12,7 +11,21 @@ const pify = require("pify");
 // fs tasks when running the custom importer code.
 // This can be removed as soon as node-sass implements a fix for this.
 const threadPoolSize = process.env.UV_THREADPOOL_SIZE || 4;
```

Comment:
Let move this variables before `asyncSassJobQueue = async.queue(sass.render, threadPoolSize - 1);`, it don't have sense in top of file

Reply:
Makes sense. I should've thought of this. 😝